doc/getting_started/installation_linux.rst

```diff
@@ -8,27 +8,20 @@ This section describes how to set up a Linux development system.
 After completing these steps, you will be able to compile and run your Zephyr
```

Comment:
Can we perhaps add a reference to `host_setup` so that users that were linked to this page directly can also go back and see the full installation picture? Also, the `after completing these step` is no longer valid, since we now have additiona instructions in the main Getting Started page.

Reply:
Nice catch, thank you. I also tested the build in a Fedora 28 Docker container, not 25, so I'll update the version number for that as well.